andela/firestar-backend
# #167730752 Fix heroku build

#### What does this PR do?
Creates no script to run build on heroku

#### Description of Task to be completed?
Create a script to run dist folder

#### How should this be manually tested?
git clone this repo using the command `git clone git@github.com:andela/firestar-backend.git`
  - cd to the project
  - checkout to the branch using the command `git checkout ft-user-roles-setup-167730752`
- Check the package.json file for the clean and build scripts

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
